Source/Orts.Simulation/Simulation/RollingStocks/SubSystems/Brakes/MSTS/AirSinglePipe.cs

```diff
@@ -137,12 +175,25 @@ public override void InitializeFromCopy(BrakeSystem copy)
         {
             AirSinglePipe thiscopy = (AirSinglePipe)copy;
             MaxCylPressurePSI = thiscopy.MaxCylPressurePSI;
+            ReferencePressurePSI = thiscopy.ReferencePressurePSI;
+            AuxResVolumeM3 = thiscopy.AuxResVolumeM3;
             AuxCylVolumeRatio = thiscopy.AuxCylVolumeRatio;
             AuxBrakeLineVolumeRatio = thiscopy.AuxBrakeLineVolumeRatio;
             EmergBrakeLineVolumeRatio = thiscopy.EmergBrakeLineVolumeRatio;
+            SupplyBrakeLineVolumeRatio = thiscopy.SupplyBrakeLineVolumeRatio;
+            CylBrakeLineVolumeRatio = thiscopy.CylBrakeLineVolumeRatio;
             EmergResVolumeM3 = thiscopy.EmergResVolumeM3;
+            SupplyResVolumeM3 = thiscopy.SupplyResVolumeM3;
             BrakePipeVolumeM3 = thiscopy.BrakePipeVolumeM3;
             CylVolumeM3 = thiscopy.CylVolumeM3;
+            TotalCylVolumeM3 = thiscopy.TotalCylVolumeM3;
+            CylPipeVolumeM3 = thiscopy.CylPipeVolumeM3;
+            CylDiameterM = thiscopy.CylDiameterM;
+            CylAreaM2 = thiscopy.CylAreaM2;
+            CylStrokeM = thiscopy.CylStrokeM;
+            CylCount = thiscopy.CylCount;
+            CylTravelTab = CylTravelTab == null ? null : new Interpolator(thiscopy.CylTravelTab);
```

Comment:
Ah good catch, this would cause the initialization to waste time recalculating the brake cylinder travel on every single instance of a train car. Not fatal, but wasteful.